Feat: Allow categories in resource table to be compressed

stampyzfanz opened the pull request.

stampyzfanz committed:
Feat: Allow categories in resource table to be compressed

spencers145 commented:
couple of things: first, there's conflicts with master that need to be resolved. Second, the color of the category headers don't update with the style of the world, so they need to be dynamic with that. Third, the size of the resource table changes frequently when opening and closing tabs, which is quite jarring. Try fixing the size of the table to match that of the longest element, even if that element is in a collapsed category. Finally, the transition from open to closed and closed to open isn't animated. If at all possible, see if there's a way to animate the transition.

stampyzfanz committed:
Merge remote-tracking branch 'remotes/upstream/master'

stampyzfanz committed:
I have to commit, the stash docs told me to

stampyzfanz committed:
I didn't fix the conflict how do u like that git (its style.min doesnt matter)

stampyzfanz committed:
Remove conflicts. Fix resource table width. Fix background color of resource heading

Fetch and merge all commits I missed
Add mininum width for resource table to stop it from pulsating

stampyzfanz commented:
It looks like git auto-updated my pr. I should have fixed everything you said, though the column size of the table is annoying some of the time.
